README.md

```diff
@@ -139,11 +139,11 @@ The following items are stored in GitHub.
 
 * Source code, System deployment scripts (Ansible), Instructions/How to Information
 
-|  Description | Storage Location | Frequency  |
-|---|---|---|
-| Jenkins (ci) - Configuration and Settings | localhost `/mnt/backup-server/jenkins_backup` | Daily |
-| Nagios - Configuration and Settings | localhost `/root/backups` | Weekly |
-| AWX - Configuration and Settings | not currently backed up | N/A |
+| Name | Description | Storage Location | Frequency | Recovery steps |
+|---|---|---|---|
```

Comment:
In the Rich Text Display, this table isn't being shown as a table. I think it may be because you need an extra `---|`